src/sandstorm/test-app/test-app.capnp

```diff
@@ -0,0 +1,82 @@
+@0xdb849475182f3bb4;
+
+#$import "/capnp/c++.capnp".namespace("sandstorm::testapp");
+
+using Powerbox = import "/sandstorm/powerbox.capnp";
+using Grain = import "/sandstorm/grain.capnp";
+using Spk = import "/sandstorm/package.capnp";
+
+const testAppHtml :Data = embed "test-app.html";
```

Comment:
I love embedding resources directly into the binary! It's so much easier to deal with and there's really no down side.